### DISCUSSION TITLE
Is there a way to partition existing chat exports? Additionally, is there a way to convert them from HTML -> JSON/CSV?

### DISCUSSION THREAD

**Discussion by yokaimotorist** [Tyrrrz/DiscordChatExporter#1154] [Q&A]
First of all, thank you for your wonderfully nifty program. It is _so_ appreciated.
These are the questions I've had. 

Thank you for reading my thread!

**Comment by 96-LB** [comment]
There is no way to do either of these with DiscordChatExporter. DCE cannot read or modify existing exports. You'd have to run your own script or an external tool to modify the files after they've been created.

**Reply by yokaimotorist** [reply]
Thank you for your answer. Do you know of any external tools that would work for this use case?

**Reply by 96-LB** [reply]
Not off the top of my head, sorry.

HTML exports also don't contain all the information that JSON exports do, so you would be missing some info.

**Comment by EDLLT** [comment]
Try this converter by Stoxis
https://github.com/Tyrrrz/DiscordChatExporter/discussions/1047#discussioncomment-6870050
